criteria/findability.md

```diff
@@ -0,0 +1,51 @@
+# Findability
+
+## Requirements
+
+* The codebase MUST be findable using a search engine by description of the problem it solves in natural language.
+* The codebase MUST be findable using a search engine by codebase name.
+* Maintainers SHOULD submit the codebase to relevant software catalogs.
+* The codebase SHOULD have a website which describes the problem the codebase solves using the preferred jargon of different potential users of the codebase (including technologists, policy experts and managers).
```

Comment:
With Open Zaak we now see that this website exists, but content-wise it's not really maintained nor clear who should maintain this. I do see the value in this, but I'm worried this will take away valuable development resources if you want to maintain this properly. So I'm not sure if this should be a SHOULD or MAY.

Reply:
That is a justified worry, especially for codebase communities that haven't grown to include several active public organizations yet (or doesn't have an active product steering team). As this happens, there is a natural tendency that not everyone can contribute through developers, but that their resources at hand might be in the communications or information departments. While they might not be very technical, they will probably be able to solve everyday tasks (of course dependent on the platform used).

Reply:
I don't believe this should be (solely) on a seperate website. I would like to see the information described in the repository itself. So you have one place that covers all necessary information to be able to use, implement and to asses the usefullnes of a solution.

Reply:
You're correct, it shouldn't be solely on a separate website and indeed we already have requirements for this to be part of the documentation of the codebase. The documentation might be too detailed for findability and perhaps also not that easy to discover through search engines (and I don't think we necessarily want to create requirements that it should be). A landing page with a clear description with links to the repository and its documentation on its own domain or even on GH pages would be better suited for this task.